pkgs/misc/drivers/epsonscan2/default.nix

```
@@ -55,26 +55,29 @@ stdenv.mkDerivation {
 
   patches = [
     ./build.patch
+    ./gcc14.patch
     (fetchpatch {
-      url = "https://github.com/flathub/net.epson.epsonscan2/raw/master/patches/epsonscan2-crash.patch";
-      hash = "sha256-srMxlFfnZuJ3ed5veFcJIiZuW27F/3xOS0yr4ywn4FI=";
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/0002-Fix-crash.patch?h=epsonscan2";
```

```suggestion
      url = "https://aur.archlinux.org/cgit/aur.git/plain/0002-Fix-crash.patch?h=epsonscan2&&id=6e422c487f5f3b135df554fde65ab57bc6377159";
```
Same for other URLs, to ensure the content doesn't change.

---

i did so, but by keeping the flatpak patch sources, as they also apply correctly on 6.7.70.0 (which i did not even check before).